### DISCUSSION TITLE
Angular 14 support?

### DISCUSSION THREAD

**Discussion by kalani96746** [angular/angularfire#3229] [General]
Its gonna be released any minute and I am hot on the trails of an upgrade all the way from Angular 9 to 13..and donʻt want to stop at 13 with 14 almost out..

But also need the angularfire support..is there a branch or project where this work is taking place?  Or do you always work from the current release branch after the fact of release?

**Comment by jamesdaniels** [comment]
We verified and marked the peer as supported in 7.4, which is being cut right as we speak #3234 

Needed a day or two to verify things were working correctly and don't like cutting on a Friday ;) If you see something let us know.
